adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -431,15 +444,22 @@ - (void)vungleWillShowAdForPlacementID:(nullable NSString *)placementID {
   if (!placementID.length) {
     return;
   }
-  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID];
+
+  id<GADMAdapterVungleDelegate> delegate = [self getDelegateForPlacement:placementID withBannerRouterDelegateState:BannerRouterDelegateStateWillPlaying];
+  // The delegate is not Interstitial or Rewarded Video Ad
+  if (!delegate && ! _bannerPlacementID) {
+    _bannerPlacementID = placementID;
+    delegate = [self getDelegateForPlacement:placementID withBannerRouterDelegateState:BannerRouterDelegateStateWillPlaying];
```

Comment:
For a refresh request Banner, after calling the callback method `-(void)vungleDidCloseAdWithViewInfo:placementID:` of the old Banner, the `_bannerPlacementID` will be set to nil, so in the callback method `-(void)vungleWillShowAdForPlacementID:` of the refresh request Banner, we will reassign `_bannerPlacementID`. Thanks. 🙂